hack/deploy/controlplane/external-etcd-patch.yaml

```diff
@@ -0,0 +1,52 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: multicluster-controlplane
+spec:
+  template:
+    spec:
+      containers:
+      - name: controlplane
+        args:
+          - "/multicluster-controlplane"
+          - "--authorization-mode=RBAC"
+          - "--enable-bootstrap-token-auth"
+          - "--service-account-key-file=/controlplane/cert/kube-serviceaccount.key"
+          - "--client-ca-file=/controlplane/cert/client-ca.crt"
+          - "--client-key-file=/controlplane/cert/client-ca.key"
+          - "--enable-bootstrap-token-auth"
+          - "--enable-priority-and-fairness=false"
+          - "--api-audiences="    
+          - "--storage-backend=etcd3"
+          - "--v=1"
+          - "--service-account-lookup=false"
+          - "--service-account-signing-key-file=/controlplane/cert/kube-serviceaccount.key"
+          - "--enable-admission-plugins=NamespaceLifecycle,ServiceAccount,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ManagedClusterMutating,ManagedClusterValidating,ManagedClusterSetBindingValidating"
+          - "--bind-address=0.0.0.0"
+          - "--secure-port=9443"
+          - "--tls-cert-file=/controlplane/cert/serving-kube-apiserver.crt"
+          - "--tls-private-key-file=/controlplane/cert/serving-kube-apiserver.key"
+          - "--feature-gates=DefaultClusterSet=true,OpenAPIV3=false"
+          - "--enable-embedded-etcd=false"
+          - "--etcd-servers=http://etcd-0.etcd.multicluster-controlplane-etcd:2379"
```

Comment:
You can specify `--etcd-servers` with all etcd headless services comma separated